.buildkite/hooks/pre-command

```diff
@@ -13,6 +13,9 @@ GITHUB_TOKEN_VAULT_PATH="kv/ci-shared/platform-ingest/github_token"
 
 if [[ "$BUILDKITE_PIPELINE_SLUG" == "beats" || "$BUILDKITE_PIPELINE_SLUG" == "filebeat" || "$BUILDKITE_PIPELINE_SLUG" == "auditbeat" || "$BUILDKITE_PIPELINE_SLUG" == "heartbeat" || "$BUILDKITE_PIPELINE_SLUG" == "deploy-k8s" ]]; then
   source .buildkite/env-scripts/env.sh
+  if [[ -z "${GO_VERSION-""}" ]]; then
```

Comment:
Lines 16-18 not needed

Reply:
no, it should be the same as it is in the main

Reply:
will fix it in one of the future PRs